pandas/tests/arrays/test_datetimes.py

```diff
@@ -179,6 +179,22 @@ def test_setitem_clears_freq(self):
         a[0] = pd.Timestamp("2000", tz="US/Central")
         assert a.freq is None
 
+    @pytest.mark.parametrize(
+        "obj",
+        [
+            pd.Timestamp.now(),
+            pd.Timestamp.now().to_datetime64(),
+            pd.Timestamp.now().to_pydatetime(),
+        ],
+    )
+    def test_setitem_objects(self, obj):
+        # make sure we accept datetime64 and datetime in addition to Timestamp
```

Comment:
this is worth a release note?

Reply:
only affects the EAa, should't be user-facing